tests/caasp/one_line_checks.pm

```diff
@@ -24,6 +24,9 @@ sub run {
     # bsc#1019652 - Check that snapper is configured
     assert_script_run "snapper list";
 
+    # https://build.opensuse.org/request/show/583954
```

Comment:
Will not work - /var/lib/cni is NOT a subvolume on Kubic/CaaSP 4 and never will be (/var is a subvolume already)